smack-websocket/src/main/java/org/jivesoftware/smack/websocket/implementations/OkHttpWebsocketImpl.java

```diff
@@ -0,0 +1,189 @@
+/**
+ *
+ * Copyright 2020 Aditya Borikar
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jivesoftware.smack.websocket.implementations;
```

Comment:
This should be in `org.jivesoftware.smack.websocket.implementations.okhttp`. Because multipole subprojects should ideally never place classes in the same package. And if we ever get another websocket impl, then we would have that case.